cg/meta/workflow/nextflow_common.py

```diff
@@ -242,3 +238,47 @@ def write_deliverables_bundle(
         WriteFile.write_file_from_content(
             content=deliverables_content, file_format=file_format, file_path=file_path
         )
+
+    @classmethod
+    def get_sbatch_path(cls, case_id: str, root_dir: str) -> Path:
+        """Returns a path where the nextflow sbatch for the head job should be located."""
+        return Path(
+            cls.get_case_path(case_id=case_id, root_dir=root_dir), "nextflow_head_job.sbatch"
+        )
+
+    @classmethod
+    def execute_head_job(
+        cls,
+        case_id: str,
+        root_dir: str,
+        slurm_account: str,
+        email: str,
+        qos: str,
+        commands: str,
+        hours: int = 96,
+        memory: int = 10,
+        number_tasks: int = 1,
```

Comment:
Would it make sense to define this as constants part of a `StrEnum` class?

Reply:
Sounds good. I'll make a class but not `StrEnum` since values are not strings.

Reply:
We also have an IntEnum class FYI :) @fevac